detect_secrets/filters/heuristic.py

```diff
@@ -113,3 +113,53 @@ def is_non_text_file(filename: str) -> bool:
         '.zip',
     ),
 )
+
+
+def is_templated_secret(secret: str) -> bool:
+    """
+    Filters secrets that are shaped like: {secret}, <secret>, or ${secret}.
+    """
+    if secret[0] == '$':
```

Comment:
Did you forget to remove this after copying it to the new `is_prefixed_with_dollar_sign ` function?

Reply:
You're absolutely right. ++